### PR TITLE
Added BlockNumber parameter to Contract query

### PR DESCRIPTION
**Pull request by usd-yamazaki**
I want to add a parameter to call constant function by specifying a block number.
Thank you.

### CODE REVIEW

File: src/contract/mod.rs [hunk 1]
```diff
@@ -126,7 +126,7 @@ impl<T: Transport> Contract<T> {
   }
 
   /// Call constant function
-  pub fn query<R, A, P>(&self, func: &str, params: P, from: A, options: Options) -> QueryResult<R, T::Out> where
+  pub fn query<R, A, P>(&self, func: &str, params: P, from: A, options: Options, block: Option<BlockNumber>) -> QueryResult<R, T::Out> where
```
Comment (tomusdrw) [inline src/contract/mod.rs]: Can you make it `B: Into<Option<BlockNumber>>`? This allows to omit `Some` when calling the function.

Comment (usd-yamazaki) [reply src/contract/mod.rs]: Thank you.
I learned a lot!

**Comment by usd-yamazaki** [comment]
I do not know why this test was canceled.

**Comment by tomusdrw** [comment]
Me neither, but went well after restart. Thanks!

**Pull request merged**